x-pack/test/common/lib/test_data_loader.ts

```diff
@@ -124,5 +125,28 @@ export function getTestDataLoader({ getService }) {
         ])
       );
     },
+
+    deleteAllSavedObjectsFromKibanaIndex: async () => {
+      await es.deleteByQuery({
+        index: '.kibana',
+        wait_for_completion: true,
+        body: {
+          conflicts: 'proceed',
+          query: {
+            bool: {
+              must_not: [
+                {
+                  term: {
+                    type: {
+                      value: 'space',
```

Comment:
Are the only 'documents' in the .kibana index spaces and saved objects?
Nit: filtering the type inclusively would be safer, but not necessary if above is true.

Reply:
There do appear to be some other types of saved objects, but they don't seem relevant to the test suite. I will make this change as a future enhancement when I port more FTR tests to the kbnImport